src/Gzero/Core/Notifications/ResetPassword.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Gzero\Core\Notifications;
```

Comment:
namespace should be in first line, after php

Reply:
ok